src/modules/ItinerarySearchResultsModule/index.tsx

```
@@ -59,15 +61,22 @@ const ItinerarySearchResultsModule = () => {
-      const responseLikes = await customFetch<BatchCheckItinerarySavedResponse>(
-        `/itineraries/checkSave`,
-        {
-          method: 'POST',
-          body: customFetchBody(itineraryIds),
-          credentials: 'include',
-        }
-      )
-      setItinerariesLiked(responseLikes.result)
+      if (!isAuthenticated) {
+        setItinerariesLiked(
+          itineraryIds.reduce((acc, id) => ({ ...acc, [id]: false }), {})
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)